job_runner/cli.py

```diff
@@ -70,6 +73,7 @@ def main(args):
 
     for job_status in job_runner.run(parsed_args.job):
         for reporter in reporters:
+            logger.info(f"Job status: {job_status}")
```

Comment:
Can you move this outside the loop `reporters` loop? I probably told you to put it here.